make/buf/scripts/createupgradepr.bash

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
```

Comment:
Make sure to apply comments from other copy of this file.

Change file name to `make/buf/scripts/githubactionmakeupgrade.bash`